setup.py

```diff
@@ -3,7 +3,7 @@
 requirements = ['Flask', 'werkzeug', 'jinja2', 'peewee>=3.0.0', 'wtforms', 'wtf-peewee']
 setup(
     name='flask-peewee',
-    version='3.0.3',
+    version='3.0.4',
```

Comment:
Does semver allow for any sort of variant specification? I guess it doesn't matter since the official fork of `flask-peewee` has been EOLed, but seems like it'd be nice to not have to worry about version collisions.